data/yam/agama/auto/lib/addons.libsonnet

```diff
@@ -0,0 +1,12 @@
+{
+  addon_ha(reg_code=''):: {
+    id: 'sle-ha',
+    registrationCode: reg_code,
+    version: '16.0',
+  },
+  addon_we(reg_code=''):: {
```

Comment:
this was an example of how can works for the future, there is not WE addon, please drop all references in the PR.

Reply:
removed all related with we.